### ISSUE TITLE
Whenever i have a custon skin on a weapon it doesnt throw

### ISSUE THREAD

**Issue by MrAlmagro** [PickleModifications/pickle_weaponthrowing#19]
If i have a custom, skin, it just floats in the air whereever it was spawned, no sense tbh.
Normal weapons work perfectly

**Comment by Sourasish7** [comment]
bro you have to add it into config file

**Comment by MrAlmagro** [comment]
yeah men im not that dumb, anyways its a skin replace, not a skin addon, so it should still be working, since the hash is the same

**Comment by ENT510** [comment]
It's not possible because the weapon object isn't created directly. Instead, createObject is used, which doesn't support adding tints, skins, or attachments for weapons
